src/makeCalculator.js

```diff
@@ -37,7 +37,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
```

Comment:
```suggestion
  const calculator = {
    result: 0,

    reset() {
      this.result = 0;

      return this;
    },

```

Reply:
agree, add some blank lines to improve readability